internal/daemonset/daemonset_test.go

```diff
@@ -24,9 +28,16 @@ const (
 	devicePluginImage = "device-plugin-image"
 )
 
+var ctrl *gomock.Controller
+
 var _ = Describe("SetDriverContainerAsDesired", func() {
+	var gomockCtrl *gomock.Controller
```

Comment:
Why do we have two `*gomock.Controller` here?

Reply:
not anymore